DEV_DOCS.md

```diff
@@ -194,9 +194,22 @@ If a translation of a particular string is not yet available, then auspice will
 
 ## Releases & versioning
 
-1. Run the `./releaseNewVersion.sh` script from an up-to-date `master` branch. It will prompt you for the version number increase, push changes to the `release` branch and, as long as the GitHub workflow is successful then a new version will be automatically published to [npm](https://www.npmjs.com/package/auspice).
-    - Ensure the [docker-base CI action triggered by nextstrain-bot](https://github.com/nextstrain/docker-base/actions/workflows/ci.yml?query=branch%3Amaster+actor%3Anextstrain-bot) runs successfully.
-2. Update the version on Bioconda:
+Releasing Auspice requires a few manual steps as there are so many downstream targets which are each to be updated with the new version.
+
+1. Double check there are entries at the top of the `CHANGELOG.md` and that they have been committed to the master branch and pushed to GitHub.
+1. Run the `./releaseNewVersion.sh` script from an up-to-date `master` branch.
+    This will prompt you for the version number increase and push changes to both the master and release branches.
+    * Auspice's [CI Action](https://github.com/nextstrain/auspice/actions/workflows/ci.yaml) will publish the new version to npm. 
+    It takes some time for the new version to appear in the npm registry.
+    To check if it has been released you can monitor the slack channel `#nextstrain-admin`, run `npm view auspice versions --json | tail` (or similar) or check [npmjs.com/package/auspice](https://www.npmjs.com/package/auspice).
+    None of the following numbered steps will work until the new version is available on npm. 
+    * The release script will automatically create an entry on [github.com/nextstrain/auspice/releases](https://github.com/nextstrain/auspice/releases/) containing information from the changelog.
+    * Ensure the [docker-base CI action triggered by nextstrain-bot](https://github.com/nextstrain/docker-base/actions/workflows/ci.yml?query=branch%3Amaster+actor%3Anextstrain-bot) runs successfully.
+    (This action in nextstrain/docker-base is triggered by Auspice's CI action.)
+1. Nextstrain.org will check daily for a new version of Auspice and [automatically merge the resulting PR](https://github.com/nextstrain/nextstrain.org/blob/175171e0e1c1b331538729a1168598227d08698d/.github/workflows/dependabot-automation.yml).
```

Comment:
Suggestion: hint at *why* nextstrain.org merges automatically (unlike auspice.us)

```suggestion
1. Nextstrain.org will check daily for a new version of Auspice and [automatically merge the resulting PR](https://github.com/nextstrain/nextstrain.org/blob/175171e0e1c1b331538729a1168598227d08698d/.github/workflows/dependabot-automation.yml). It should show up on https://next.nextstrain.org automatically.
```

Reply:
Thanks - updated text & added a line about promoting it as well